frontend-instructor/src/components/signup/SignUp.jsx

```
@@ -0,0 +1,179 @@
+
+  return (
+    <div className="flex flex-col items-center justify-center mt-10">
+      <div className="w-full max-w-md bg-white p-6 rounded-lg shadow-lg">
+        <h2 className="text-xl font-semibold mb-4">Instructor Sign Up</h2>
+        <p className="text-sm text-gray-500 mb-4">
+          Enter details to create an instructor account
+        </p>
+
+        <form onSubmit={signupFormik.handleSubmit} className="space-y-4">
+          <div>
+            <label className="block text-sm font-medium">Name</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)